third_party/bazel/cuda_static/cuda_static_configure.bzl

```diff
@@ -2,16 +2,20 @@ load("//bazel:common.bzl", "files_exist", "get_bash_bin")
 
 _TF_CUDA_HOME = "TF_CUDA_HOME"
 
+def _create_dummy_repository(repo_ctx):
+    repo_ctx.symlink(Label("//bazel/cuda_static:BUILD"), "BUILD")
+    repo_ctx.symlink(Label("//bazel/cuda_static:build_defs.bzl.dummy"), "build_defs.bzl")
+
 def _impl(repo_ctx):
     cuda_path = repo_ctx.os.environ.get(_TF_CUDA_HOME, None)
-    if cuda_path == None:
-        fail("Please make sure that the cuda library path is set via env var: {}".format(_TF_CUDA_HOME))
-
-    repo_ctx.symlink(Label("//bazel/cuda_static:cuda_static.BUILD.tpl"), "BUILD")
-    repo_ctx.symlink(cuda_path + "/lib64", "lib64")
-    repo_ctx.template("build_defs.bzl", Label("//bazel/cuda_static:build_defs.bzl.tpl"), {
-        "%{IF_HAS_CUBLASLT}": "True" if files_exist(repo_ctx, ["lib64/libcublasLt_static.a"], get_bash_bin(repo_ctx))[0] else "False",
-    })
+    if cuda_path != None:
+        repo_ctx.symlink(Label("//bazel/cuda_static:cuda_static.BUILD.tpl"), "BUILD")
+        repo_ctx.symlink(cuda_path + "/lib64", "lib64")
+        repo_ctx.template("build_defs.bzl", Label("//bazel/cuda_static:build_defs.bzl.tpl"), {
+            "%{IF_HAS_CUBLASLT}": "True" if files_exist(repo_ctx, ["lib64/libcublasLt_static.a"], get_bash_bin(repo_ctx))[0] else "False",
```

Comment:
No need to pass `get_bash_bin` ? it's the default: 
https://github.com/alibaba/BladeDISC/blob/17af6a344d14eaaa3f0146a451f613e2b846f56e/third_party/bazel/common.bzl#L268-L269

Reply:
removed